.gitmodules

```diff
@@ -31,3 +31,12 @@
 [submodule "external/alfons"]
 	path = external/alfons
 	url = https://github.com/hjanetzek/alfons
+[submodule "external/curl"]
+	path = external/curl
+	url = https://github.com/lygstate/curl.git
+[submodule "external/zlib"]
+	path = external/zlib
+	url = https://github.com/madler/zlib.git
+[submodule "external/angle"]
+	path = external/angle
+	url = https://github.com/lygstate/angle.git
```

Comment:
@hjanetzek  curl support for using Windows's default SSL support. WinSSL.
About using 
```
git submodule update --init external/zlib
git submodule update --init external/curl
git submodule update --init external/angle
```
It's would increase maintenance burden . For example, how to specify which version we should use?